.babelrc

```diff
@@ -1,19 +1,39 @@
 {
   "sourceMaps": true,
   "presets": [
-    "es2015",
-    "stage-2"
+    "@babel/preset-env",
+    "@babel/preset-flow"
   ],
   "plugins": [
     ["add-module-exports"],
-    ["transform-es2015-modules-umd"],
-    ["transform-react-jsx", { "pragma": "h" }],
-    ["transform-runtime", {
-      "helpers": false,
-      "polyfill": true,
-      "regenerator": true,
-      "moduleName": "babel-runtime"
-    }],
-    [ "transform-flow-strip-types" ]
+    ["@babel/plugin-transform-modules-umd"],
+    [
+      "@babel/plugin-transform-react-jsx",
+      {
+        "pragma": "h"
+      }
+    ],
+    [
+      "@babel/plugin-transform-runtime",
+      {
+        "corejs": 2
+      }
+    ],
+    ["@babel/plugin-transform-flow-strip-types"],
+    "@babel/plugin-syntax-dynamic-import",
```

Comment:
A few new plugins here now. Is there anything actually new being added? Or are these all just part of babel's effort to be more explicit?

Reply:
babel effort to be more explicit, they got rid of stage refs. I have added one on exports though.